issue_03/script.js

```diff
@@ -0,0 +1,16 @@
+const ul = document.getElementById('js-ul');
+const hrefArray = ['a1.html', 'a2.html'];
+const textArray = ['a1', 'a2'];
+const srcArray = ['/img/bookmark.png', '/img/message.png'];
+
+for (let i = 0; i < 2; i++) {
+    const li = document.createElement('li');
+    const a = document.createElement('a');
+    const img = document.createElement('img');
+
+    a.setAttribute('href', hrefArray[i]);
+    a.textContent = textArray[i];
+    img.setAttribute('src', srcArray[i]);
+    a.insertAdjacentElement('afterbegin', img);
+    ul.appendChild(li).appendChild(a);
+}
```

Comment:
IMO
Using appendchild in a for loop is not good for performance.
You can study "createDocumentFragment" here! This is good study so please refer the Monaka san's article!

https://monatamalog.com/archives/291

Reply:
I see that the performance is not degraded by having only one reflow for batch insertion.
I learned a lot!!!
Thank you very much :raised_hands: